airflow/www/templates/airflow/dag.html

```diff
@@ -435,6 +441,45 @@ <h4 class="modal-title" id="dagModalLabel">
           </li>`
         );
       }
+
+      if (extra_links && extra_links.length > 0){
+        var markupArr = [];
+
+        $.each(extra_links, function(i, link){
+          var url = "{{ url_for('Airflow.extra_links') }}" +
```

Comment:
You should avoid Jinja in Javascript. I will correct it in my PR.
Reference: 
https://github.com/apache/airflow/pull/4787